sdks/java/testing/test-utils/src/main/java/org/apache/beam/sdk/testutils/Whitebox.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.testutils;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+/** Internal copy of Whitebox implementation of Mockito. */
+public class Whitebox {
```

Comment:
Let's just have one copy of this, can we? Or can we simply not use it in some of the places?

Reply:
They were in separate gradle subprojects. E.g. `Flink` would then have to depend on `sdks`. Is that desirable?